app/deployments/tests/test_tasks.py

```diff
@@ -150,3 +151,26 @@ def test_500_response_doesnt_explode(self):
         dataset = ErddapDataset.objects.get(name="N01_accelerometer_all")
 
         tasks.refresh_dataset(dataset.id)
+
+    @my_vcr.use_cassette("500_end_time.yaml")
+    def test_500_end_time(self):
+        j03 = Platform.objects.get(name="J03")
+        dataset = ErddapDataset.objects.create(
+            name="J03_aanderaa_all", server=self.erddap
+        )
+        ts = TimeSeries.objects.create(
+            platform=j03,
+            data_type=DataType.objects.get(standard_name="sea_water_salinity"),
+            variable="salinity",
+            constraints={},
+            start_time="2018-07-17 17:00:00+00",
+            dataset=dataset,
+        )
+
+        ts.refresh_from_db()
+
+        assert ts.end_time is None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/gmri/buoy_barn/pullRequest?prid=6285790)